extensions/fairifier-rdf-extension/dtls/fairifier/RdfSkeletonService.java

```diff
@@ -13,8 +14,16 @@
  */
 
 interface RdfSkeletonService {
```

Comment:
This interface could use some class level metadata (as I've explained in Monday's presentation).